packages/go/analysis/ad/ad.go

```diff
@@ -516,6 +516,8 @@ func GetRelayTargets(ctx context.Context, db graph.Database, edge *graph.Relatio
 			nodeSet, err = GetVulnerableDomainControllersForRelayNTLMtoLDAP(ctx, db, edge)
 		case ad.CoerceAndRelayNTLMToLDAPS:
 			nodeSet, err = GetVulnerableDomainControllersForRelayNTLMtoLDAPS(ctx, db, edge)
+		case ad.CoerceAndRelayNTLMToADCS:
```

Comment:
With moving ADCS to Relay Targets, do we need the case in `GetEdgeCompositionPath` anymore for ADCS?

Reply:
ADCS has both relay and composition